app/javascript/packages/document-capture/components/form-steps.jsx

```diff
@@ -115,10 +115,10 @@ function FormSteps({ steps = [], onComplete = () => {}, initialValues = {}, auto
   /**
    * Returns array of form errors for the current set of values.
    *
-   * @return {FormStepError<Record<string,Error>>[]}
+   * @return {FormStepError<Record<string,Error>>[]|undefined}
```

Comment:
What if we did a length check in the caller instead? Then it can always return an array?

Well I guess doing it this way forces a type check in all the call sites? But even nullable array type doesn't stop from possibly returning an empty array in the future right?

Reply:
Yeah, in https://github.com/18F/identity-idp/pull/4243#discussion_r496811044 I'd contemplate this, or even a combination of all of the possible remedies. It wouldn't hurt to also have a length check, so I could add it.

The idea here was to try to reinforce the point that if `activeErrors` is anything other than `undefined`, it should be assumed to be non-empty.

Another option may even be to _always_ force `activeErrors` to be an array, so we don't have to bother with considering `undefined` and just assume emptiness as absence of errors. In retrospect, this seems like a good idea, though I've not yet dug into it enough to know if there's complications in implementing it.

At the very least, the new tests should help ensure the intended behavior regardless of the specific implementation.

Reply:
> Another option may even be to _always_ force `activeErrors` to be an array, so we don't have to bother with considering `undefined` and just assume emptiness as absence of errors. In retrospect, this seems like a good idea, though I've not yet dug into it enough to know if there's complications in implementing it.

Yeah that's what I meant -- have it always return an array

But yes agree, tests are better for now

Reply:
> Yeah that's what I meant -- have it always return an array

Ah, I think I'd misread your comment the first time.

It does seem to work better this way 👍  Updated in 4aa3d23.